src/cart/crt_tree.c

```diff
@@ -230,10 +224,14 @@ crt_tree_get_children(struct crt_grp_priv *grp_priv, uint32_t grp_ver,
 	D_ASSERT(default_grp_priv != NULL);
 
 	D_RWLOCK_RDLOCK(grp_priv->gp_rwlock_ft);
-	if (ver_match != NULL)
+	if (ver_match != NULL) {
 		*ver_match = (grp_ver == default_grp_priv->gp_membs_ver ?
```

Comment:
This is weird code, why are you using a trigraph when casting to bool would be clearer?

Reply:
I agree, should be just cast to bool. I didn't add this code though, just extended the if clause. Will change never the less.